File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.c
int arraySign(int* nums, int numsSize){
    int count = 0;
    for(int i = 0; i < numsSize; i++){
        if(!nums[i]) return 0;
        else if(nums[i] < 0) count++;
    }
    return (count % 2 == 0 ? 1 : -1);
}

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.java
class Solution {
    public int arraySign(int[] nums) {
        int val = 1;
        
        for(int i = 0; i < nums.length; i++){
            if(nums[i] == 0) return 0;
            else if(nums[i] < 0) val *= -1;
        }
        return val;
    }
}